### PR TITLE
Handlebar templates

### PR DESCRIPTION
**Pull request by tyrokode**
Added Sign In and Register Page.
Sign in and Sign Out scripts.

### CODE REVIEW

**Review by breakfireaus — CHANGES_REQUESTED** [review]
please close your #if  handlebar not closed on line 42 main.handlebars

File: views/layouts/main.handlebars [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+                    </div>
+                </div>
+            </div>
+        </section>
+    </main>
+
+    {{#if signed_in}}
+    <!-- link sign-out script -->
+    <script src="/public/js/signout.js"> </script>
+
```
Comment (breakfireaus) [inline views/layouts/main.handlebars:69]: need to close the if

**Review by breakfireaus — CHANGES_REQUESTED** [review]

File: views/layouts/main.handlebars (hunk 1 above)
Comment (tyrokode) [reply views/layouts/main.handlebars:69]: Done.

**Review by Auralise — APPROVED** [review]
Looks good to me, this is a good start, we will need to possibly restructure as we increase the complete

**Comment by tyrokode** [comment]
Added Dashboard + booking templates.

**Review by breakfireaus — APPROVED** [review]
looks good

**Pull request merged**